dicetable.py

```diff
@@ -0,0 +1,327 @@
+
+
+class dicetable(object):
+    def __init__(self,dsize):
+        self.totaldice = 0
+        self.table = {0:1}
+        self.dsize = dsize
+        
+
+    def getTableVal(self,value):
+        return self.table.get(value,0)
+    def getTable(self):
+        return self.table
+    def getDsize(self):
+        return self.dsize
+    def getTotalDice(self):
+        '''returns how many dice were used to generate the values in your table'''
+        return self.totaldice
+    def max(self):
+        '''returns the highest die roll in table'''
+        return self.dsize*self.totaldice
+    def min(self):
+        '''returns the lowest die roll in table'''
+        return self.totaldice
+    
+    def addADie(self):
```

Comment:
I assume this is fine, I don't feel like trying to understand the algorithm side of this. It looks legit-ish :green_heart: